api/src/migrations/main/1639180924469-forestClientTypesNonCNonI.js

```diff
@@ -576,7 +576,7 @@ module.exports = class forestClientTypesNonCNonI1639180924469 {
         ('00012752', 'REGIONAL DISTRICT OF EAST KOOTENAY', CURRENT_USER),
         ('00012779', 'REID, COLLINS AND ASSOCIATES LTD.', CURRENT_USER),
         ('00012786', 'HORTH FORESTRY LIMITED', CURRENT_USER),
-        ('00012797', 'MINISTRY OF FORESTS, LANDS & NATURAL RESOURCE OPERATIONS', CURRENT_USER),
+        ('00012797', 'MINISTRY OF LAND, WATER and RESOURCE STEWARDSHIP', CURRENT_USER),
```

Comment:
I wasn't expecting these to change, technically we pull this data from the Client table from "THE" Oracle database, and that should be the source of truth. However, I see no reason not to make this change. (In other words, this is good.) But we should probably let the team maintaining the source data know that we expect the ministry name change to be applied.

Reply:
However, these file changes won't actually change the data in existing FOM databases (e.g. in test, prod, and demo environments). To do that, you need to create a new migration that updates the existing rows needing to be changed.

Reply:
Just now saw your original question. Strictly speaking, Julius has only asked for a change to the UI screens, and I don't see a situation in which these forest client records for the ministry will actually be used, so we could most likely just ignore the existing forest client data.

Reply:
How about popping into Postgres and making the correction manually?  Bad idea?

No idea about the source of truth team.  Do we have a contact?

Reply:
I'm shocked as a devops person you'd suggest manually :). (Yes, its a bad idea.)
Anne-Marie knows who to contact, I believe.

Reply:
Ouch.  :D  ...Thanks, I'll look in with Anne-Marie.

Reply:
@basilv You're right.  The migration changes have been dropped.  Simpler now!  Review again please?